xbmc/cores/VideoPlayer/DVDSubtitles/SubtitlesStyle.h

```diff
@@ -115,15 +115,20 @@ struct renderOpts
 {
   float frameWidth;
   float frameHeight;
+  // Video size width, may be influenced by video settings (e.g. zoom)
   float videoWidth;
+  // Video size height, may be influenced by video settings (e.g. zoom)
   float videoHeight;
   float sourceWidth;
   float sourceHeight;
+  float m_par; // Set the pixel aspect ratio
   MarginsMode marginsMode = MarginsMode::DEFAULT;
   // Vertical line position of subtitles in percentage
   // only for bottom alignment, 0 = bottom (no change), 100 = on top
   double position = 0;
   HorizontalAlign horizontalAlignment = HorizontalAlign::DISABLED;
+  // Make font size relative to the window instead of the video
+  bool m_fontScaleWithWindow = false;
```

Comment:
Is this one really needed? Maybe checking marginsmode against `SUBTITLES::STYLE::MarginsMode::INSIDE_VIDEO` could be enough?

Reply:
yes atm can be an alternative, but i think that linking font scaling to marginmode is not so good idea

Reply:
Sure, my point/request here was to prefer avoiding random bools for every single option in favor of something that can be either derived from other properties or grouped into some enum class or so.